config/gulp/scripts.js

```diff
@@ -34,12 +34,9 @@ function copyJquery() {
  */
 function compile() {
   // Stream images is imported by itself in `content/images/_index.md`.
-  return src(
-    [`${PROJECT_JS_SRC}/*.js`, `!${PROJECT_JS_SRC}/stream-images.js`],
-    {
-      sourcemaps: true,
-    }
-  )
+  return src([`${PROJECT_JS_SRC}/*.js`], {
+    sourcemaps: true,
+  })
```

Comment:
We need to remove the exclusion of `stream-images.js` in the compile function even though it is loaded in `content/images/_index.md`. But there are still some issues in the `stream-images.js` that need to be researched with the new es-lint rules.